README.md

```diff
@@ -1,12 +1,48 @@
 ![NodeSchool Montréal Logo](http://nodeschool.io/montreal/i/NodeSchoolMTL.svg)
 
-# Develop NodeSchoolMTL
+# NodeSchool Montreal Website
 
 We use [Jekyll](https://jekyllrb.com/) to build [http://nodeschool.io/montreal](http://nodeschool.io/montreal). If you want to run the site or develop it locally you'll have to install [Jekyll](https://github.com/jekyll/jekyll) and [Jekyll-Sitemap](https://github.com/jekyll/jekyll-sitemap).
 
-`gem install jekyll`
+Aditionally, in order to suport both French and English
+content, we need to use gems (Ruby modules) that are not
+supported by GitHub's gh-pages Jekyll scripts. For this
+reason we run jekyll with the help of [Travis](https://travis-ci.com/).
 
-`gem install jekyll-sitemap`
+Notes to Mentors
+----------------
+
+If you are adding yourself or removing yourself as a 
+mentor, you should create a PR request, first having modified
```

Comment:
Against which branch?

Reply:
That was stated in a section below, but we can put it in this section.

Reply:
Noticed that. This is fine.